org.mintguide.WebAppReader.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: org.mintguide.WebAppReader
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: webappreader.sh
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home  # to be able to save html pages to a hard drive
+  - --share=network # it is necessary to download html from the Internet
+  - --device=dri # For the Qt webengine module to work perfectly with gpu
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess # necessary for QtWebEngineProcess to work
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - python3-requirements.json
+  
+  - name: wget2 #  proper workable version = 2.1
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap
+      - ./configure --prefix=/app
+      - make
+      - make install
+    sources:
+      - type: git
+        url: https://gitlab.com/gnuwget/wget2.git
+        commit: 657e3d22c488a2d1f173e208e253995e15f8b6db
+        
+  - name: webappreader
+    buildsystem: simple
+    build-commands: 
+      - mkdir -p /app/webappreader/
+      - cp -r database /app/webappreader/database
+      - cp -r pages /app/webappreader/pages
+      - cp -r src /app/webappreader/src
+      - cp -r ui /app/webappreader/ui
+      - install -Dm644 constants.py -t /app/webappreader
+      - install -Dm644 settings.py -t /app/webappreader
+      - install -Dm644 tools.py -t /app/webappreader
+      - install -Dm644 webappreader.py -t /app/webappreader
+    sources:
+      - type: git
+        url: https://github.com/ShvonderSiN/WebAppReader.git
+        commit: b417286898a3cce86ed844f53368cf36db8385d3
+      - type: file
+        path: org.mintguide.WebAppReader.desktop
+      - type: script
+        commands:
+          - 'python3 /app/webappreader/webappreader.py "$@"'
+        dest-filename: webappreader.sh
+      - type: file
+        path: org.mintguide.WebAppReader.metainfo.xml
+    post-install:
+      - install -Dm644 src/logo.png /app/share/icons/hicolor/128x128/apps/org.mintguide.WebAppReader.png
```

Comment:
the image is 80x80 which is:

1. too small
2. not 128x128

Reply:
right, hope will find another sizes